### DISCUSSION TITLE
Do i have to adjust the histogram of every images selected for time range?

### DISCUSSION THREAD

**Discussion by claudioasn** [kvos/CoastSat#483] [Q&A] [answered]
Hi, my question is focused on the necessity (or not) to adjust the histogram threshold to define the shoreline position. If, for example, I have 300 images selected, I need to adjust it in ALL 300 images? How do you deal with this? Maybe would be better to train the classifier on my data?

**Comment by kvos** [comment, marked as answer]
that is because you selected settings['adjust_detection'] = True. If you set it to False you won't have to adjust the threshold.
